src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -108,6 +122,24 @@ public ConfigureOptions(
         [Option("runtime.cache.ttl-seconds", Required = false, HelpText = "Customize the DAB cache's global default time to live in seconds. Default: 5 seconds (Integer).")]
         public int? RuntimeCacheTTL { get; }
 
+        [Option("runtime.host.mode", Required = false, HelpText = "Set the host running mode of DAB in Development or Production. Default: Development.")]
+        public HostMode? RuntimeHostMode { get; }
+
+        [Option("runtime.host.cors.origins", Required = false, HelpText = "Overwrite CORS with allowed origins that are allowed to access a resource on the server. Default: [] (Array of strings).")]
```

Comment:
I did not understand what `Overwrite CORS with allowed origins that are allowed to access a resource on the server. Default: [] (Array of strings)` means, maybe you could change it a bit so it is generally more understandable.

Reply:
Overwrite Allowed Origins in CORS. Default: [] (Array of strings)